backend/sse/src/server.ts

```
@@ -0,0 +1,197 @@
+    oneofs: true,
+});
+
+const hiddenTestProcessProto: any =
+    grpc.loadPackageDefinition(packageDefinition).hidden_test_process;
+
+/**
+ * Implement the streamHiddenTestProcess
+ */
+const redis = new Redis({
+    host: process.env.REDIS_HOST,
+    port: parseInt(process.env.REDIS_PORT || "6379"),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)